include/clap/ext/params.h

```diff
@@ -195,6 +195,11 @@ enum {
    // A simple example would be a DC Offset, changing it will change the output signal and must be
    // processed.
    CLAP_PARAM_REQUIRES_PROCESS = 1 << 15,
+
+   // This parameter represents an enumerated value.
+   // If you set this flag, then you must set CLAP_PARAM_IS_STEPPED too.
+   // All values from min to max must not have a blank value_to_text().
+   CLAP_PARAM_IS_ENUM = 1 << 16,
```

Comment:
```suggestion

   // This parameter represents an enumerated value.
   // All values from min to max must not have a blank value_to_text().
   CLAP_PARAM_IS_ENUM = CLAP_PARAM_IS_STEPPED | (1 << 16),
```
Maybe we should express programmatically that enum parameters must also be CLAP_PARAM_IS_STEPPED?

Reply:
I think that wouldn't prevent programmers from manually assigning `CLAP_PARAM_IS_ENUM & ~(CLAP_PARAM_IS_STEPPED)`. So it still is important to clearly document the correct behaviour vs. only doing it programmatically. Also would be a breaking change as existing checks of `flags & CLAP_PARAM_IS_ENUM` would then also return true if only `CLAP_PARAM_IS_STEPPED` is present in `flags`

Reply:
No this is dodgy we've tried that in the past.
The problem is that `(flags & CLAP_PARAM_IS_ENUM) != 0` if flags is only stepped.

Reply:
I am fine with `CLAP_PARAM_IS_STEPPED` is implied, when `CLAP_PARAM_IS_ENUM` is set. Stepped does not mean Enum, but Enum just implies it. And the helpers could even check for `CLAP_PARAM_IS_STEPPED` being set when `CLAP_PARAM_IS_ENUM` is set, too. 

We should set one of the flags having more than one bit, this will add confusion and does not match a good code style.